tests/package.json

```diff
@@ -31,7 +31,7 @@
   "scripts": {
     "test-with-logs": "mocha --printlogs -r ts-node/register 'tests/**/test-*.ts'",
     "pre-build-contracts": "ts-node ./pre-build-contracts.ts && npx prettier -w ./contracts/compiled/*.json",
-    "test": "mocha --parallel -r ts-node/register 'tests/**/test-*.ts'",
```

Comment:
This will affect how it runs in CI also right? Do we want that? I'm okay with it.

Reply:
That command is actually not used in the CI